package.json

```diff
@@ -25,7 +25,7 @@
     "test:dependencies": "npx dependency-check@3.4.1 ./package.json",
     "test:e2e": "./scripts/killApp.sh && npm run build && run-p --race start cypress",
     "test:e2e:interactive": "./scripts/killApp.sh && npm run build && run-p --race start cypress:interactive",
-    "test:lint": "eslint --ext .js,jsx,json ./src ./data ./cypress ./.storybook ./webpack** && swagger-cli validate data/schema.yaml && stylelint 'src/**/*.js' 'src/**/*.jsx'",
+    "test:lint": "eslint --fix --ext .js,jsx,json ./src ./data ./cypress ./.storybook ./webpack** && swagger-cli validate data/schema.yaml && stylelint 'src/**/*.js' 'src/**/*.jsx'",
```

Comment:
`--fix`, It was agreed sometime ago (before your time) that we did not want the `--fix` option committed you can use this locally though.